src/main/kotlin/lotto/view/ResultView.kt

```diff
@@ -0,0 +1,38 @@
+package lotto.view
+
+object ResultView {
+    private const val LOTTO_COUNT_MESSAGE = "개를 구매했습니다."
+    private const val LOTTO_NUMBER_DELIMITER = ", "
+    private const val LOTTO_NUMBER_PREFIX = "["
+    private const val LOTTO_NUMBER_SUFFIX = "]"
+    private const val LOTTO_RESULT_MESSAGE = "당첨 통계"
+    private const val LOTTO_RESULT_DELIMITER = " - "
+    private const val LOTTO_RESULT_MATCH_MESSAGE = "개 일치"
+    private const val LOTTO_RESULT_WINNING_MONEY_TERMS = "원"
+    private const val LOTTO_RESULT_WINNING_MONEY_PREFIX = "("
+    private const val LOTTO_RESULT_WINNING_MONEY_SUFFIX = ")"
+    private const val LOTTO_RESULT_COUNT_MESSAGE = "개"
+    private const val LOTTO_RESULT_REVENUE_MESSAGE_PREFIX = "총 수익률은 "
+    private const val LOTTO_RESULT_REVENUE_MESSAGE_SUFFIX = "입니다."
+    private const val LOTTO_RESULT_REVENUE_LOSS_MESSAGE = "(기준이 1이기 때문에 결과적으로 손해라는 의미임)"
+
+    fun printLottoCount(count: Int) {
+        println(count.toString() + LOTTO_COUNT_MESSAGE)
```

Comment:
다음과 같이 작성하면 count 값을 string 으로 캐스팅 할 필요 없을 거 같습니다!
```suggestion
        println("$count$LOTTO_COUNT_MESSAGE")
```

Reply:
네, 꿀 팁 감사합니다! 추후에도 비슷한 경우가 발생하면 `$` 키워드를 사용하여 불필요한 캐스팅을 하지 않도록 하겠습니다 😃